src/main/java/pl/lukakan/contactform/MessageRepository.java

```diff
@@ -0,0 +1,28 @@
+package pl.lukakan.contactform;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+import org.thymeleaf.TemplateEngine;
+
+@Repository
+public class MessageRepository {
```

Comment:
`Repository` to kontener na dane, w związku z czym nazwa ta średnio pasuje do tej klasy (tak samo jak adnotacja @Repository), mozna ją zmienić na np.`MessageCreator`